src/scenario.h

```diff
@@ -89,6 +100,8 @@ typedef struct {
 	sint32 flags;				// 0x0268
 	uint32 company_value;		// 0x026C
 	char completed_by[64];		// 0x0270
+	scenario_source source_game;// new in OpenRCT2
```

Comment:
We typically don't define occurrences of enums, the enum itself isn't specifying a type size and you might not always want it to use the default. uint8 would be better.